src/Audio/babylon.sound.ts

```diff
@@ -98,11 +98,25 @@ module BABYLON {
                         if (typeof (urlOrArrayBuffer) === "string") this._urlType = "String";
                         if (Array.isArray(urlOrArrayBuffer)) this._urlType = "Array";
                         if (urlOrArrayBuffer instanceof ArrayBuffer) this._urlType = "ArrayBuffer";
+                        if (urlOrArrayBuffer instanceof MediaStream) this._urlType = "MediaStream";

```

Comment:
should use else here

Reply:
I don't agree. If we remove the `if` and replace it with an `else` statement that would make `_urlType = MediaStream` in the case of not handled types like `boolean`.

By default `_urlType` is `"Unknown"`. That counts for `else`

Reply:
Ohh. Should I add that to the rest of the `if` statements too?

Reply:
Yep please:)